src/components/Collections.jsx

```diff
@@ -0,0 +1,110 @@
+import { useEffect, useState } from "react";
```

Comment:
Adhered well to the proper coding style guidelines like Airbnb, bootstrap and cleaned code guidance.
The structure of the code is in compliant with coding semantic properly.
The code and view is properly getting resized based on the window size.

Reply:
Thank you!